omopetl/transform.py

```diff
@@ -155,7 +156,25 @@ def transform_normalize_date(self, current_data, target_column, transformation):
         """Normalize date values to a specific format."""
         source_column = transformation.get("source_column")
         date_format = transformation.get("format", "%Y-%m-%d")
-        return pd.to_datetime(current_data[source_column], errors="coerce").dt.strftime(date_format)
+        return pd.to_datetime(self.data[source_column], errors="coerce").dt.strftime(date_format)
+
+    def transform_compare_deathtime(self, source_column, target_column, transformation):
```

Comment:
The `transform_*` methods are a set of general functions for transforming data from one column to another. Does `transform_compare_deathtime` fit here?